src/js/sdk.tsx

```diff
@@ -40,6 +40,7 @@ const DEFAULT_OPTIONS: ReactNativeOptions = {
   enableOutOfMemoryTracking: true,
   patchGlobalPromise: true,
   transportOptions: {
+    bufferSize: DEFAULT_BUFFER_SIZE,
```

Comment:
It's ok to call it `bufferSize` here since it inherits from JS.
But for RN Apps, lets stick to `maxQueueSize` high level property, so one only does `options.maxQueueSize = X`.

Reply:
Relates to https://github.com/getsentry/sentry-react-native/pull/2578/files#r1007056274

Reply:
I can add `maxQueueSize` but then we will be advertising two places where people can set the `bufferSize`/`maxQueueSize`, because as you said we inherit it from JS. That could be super confusing. Especially given only one option will work.

I would say if we want to match the mobile SDKs that is fine, but then we should break the inheritance of the options from the JS SDK so we can remove `bufferSize` from the `transportOptions`.